builtin/trap_osh.py

```diff
@@ -152,6 +153,15 @@ def ThisProcessHasTraps(self):
         return len(self.traps) != 0 or len(self.hooks) != 0
 
 
+def _IsUnsignedInteger(s):
+    # type: (str) -> bool
+
+    try:
+        intval = mops.FromStr(s)
+    except ValueError:
+        return False
+    return not mops.Greater(mops.ZERO, intval)
```

Comment:
I think this would need `mops.Equal(mops.ZERO) or` ...  for `trap 0 INT`

Or probably a better idea is to do something like

```
util.RegexSearch('^[0-9]+$')
```

we could also use something like `match.LooksLikeInteger()`, without the negative number check, but that is a more fiddly change

Reply:
The Regex search is probably better, but I believe the current version is correct because I use NOT(GREATER(x,y)) which is the same as GREATER_OR_EQUAL(y,x).